R/val_prefix.R

```diff
@@ -13,12 +13,14 @@
 #' prefix(rnorm(10, 10), "*", format = "d")
 #' prefix(percent(c(0.1, 0.25)), ">")
 prefix <- function(x, prefix = "", sep = "", ..., na.text = NULL) {
-  formattable(x, ...,
+  formattable(x,
+    class = "formattable_prefix",
```

Comment:
Is there a reason `prefix` and `suffix` uses a class? These two functions are intended to tweak existing formats.

I inspect the `class` attribute like the following:

```r
> str(suffix(prefix(currency(acct, symbol = "HK$"), "*"), "#"))                                                                                                                                                                                                                                                                          
 'formattable_accounting' num [1:3] *HK$10,000.00# *HK$500,000.00# *HK$-20,000.00#
 - attr(*, "formattable")=List of 5
  ..$ formatter: chr "formatC"
  ..$ format   :List of 3
  .. ..$ format  : chr "f"
  .. ..$ big.mark: chr ","
  .. ..$ digits  : int 2
  ..$ preproc  : NULL
  ..$ postproc :List of 3
  .. ..$ :function (str, x)  
  .. ..$ :function (str, x)  
  .. ..$ :function (str, x)  
  ..$ class    : chr [1:4] "formattable_suffix" "formattable_accounting" "formattable" "numeric"
```

However, the `prefix` class seems missing somehow?

Reply:
Good point, this doesn't look right. Maybe prefix and suffix don't need their own class.